doc/distributed-tracing-for-rust-service-clients.md

```diff
@@ -412,3 +412,83 @@ pub fn get_operation_templates_lro_client(&self) -> OperationTemplatesLroClient
 ```
 
 This adds a clone of the parent client's `tracer` to the subclient - it functions similarly to `#[tracing::new]` but for subclient instantiation.
+
+## Verifying distributed tracing support
+
+The `azure_core_test::tracing` package provides functionality to allow developers to verify that their distributed tracing functionality is generating the expected tracing information.
+
+This functionality is driven from the `azure_core_test::tracing::assert_instrumentation_information` API.
+
+This function takes two closures and a structure which describes the expected API shape. The first closure is used to create an instance of the public API, the second actually executes the public API.
+
+As an example, here is a test for the `keyvault_secrets` package:
+
+```rust
```

Comment:
Probably should have a module import this with a `doctest` condition to make sure it's all correct and stays correct.

Reply:
Never mind. That's not going to work. I tried a few things, but the way doc examples work everything has to be in the example, and the `#` prefix doesn't work with pure markdown (still renders). Not the end of the world.